cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -214,31 +214,35 @@ private static Integer getResourceRequirement(ResourceRequirements resources, Re
         return null;
     }
 
-    private static CpuCapacity getCpuBasedOnRequirements(ResourceRequirements resourceRequirements) {
+    static CpuCapacity getCpuBasedOnRequirements(ResourceRequirements resourceRequirements) {
```

Comment:
Why is this not private anymore?

Reply:
It must have been a mistake. Private added again